tests/ml/ml_tools/generateunittests.py

```diff
@@ -89,7 +89,7 @@ def to_cpp(ndarray):
     load_timer.start();
 
     Opm::ML::NNModel<Evaluation> model;
-    OPM_ERROR_IF(!model.loadModel("%s"), "Failed to load model");
+    OPM_ERROR_IF(!model.loadModel(std::filesystem::current_path() / "%s"), "Failed to load model");
```

Comment:
tests run in the 'tests' folder in the build folder. in there, any data marked as TEST_DATA_FILES in CMakeLists.txt is mirrored, without the tests/ prefix, ie. you want ml/ml_tools/.. below.